api/src/main/java/org/apache/brooklyn/api/objs/BrooklynObject.java

```diff
@@ -141,4 +149,11 @@
          */
         boolean unsubscribe(SubscriptionHandle handle);
     }
+    
+    public interface RelationSupport<T extends BrooklynObject> {
```

Comment:
Recently added *Support interfaces implement `Iterable`, see https://github.com/apache/incubator-brooklyn/pull/911 for discussion.